### ISSUE TITLE
Requirement of a note for clarification

### ISSUE THREAD

**Issue by samipak458** [microsoft/studentambassadors#15]
## Describe the bug
Bug on [Tutorial publish a GatsBy site](https://docs.microsoft.com/en-us/azure/static-web-apps/publish-gatsby)
After the 8th point, under the heading Deploy your Web App, there is a need for a note which clarifies the most common issue/error that the user can't find the organization or repository.

## To Reproduce
**Note:**
If you don't see any repositories, you may need to authorize Azure Static Web Apps on GitHub. Then browse to your GitHub repository and go to **Settings > Applications > Authorized OAuth Apps**, select **Azure Static Web Apps**, and then select **Grant**. For organization repositories, you must be an owner of the organization to grant the permissions.

## Expected behavior
While following the tutorial, some users might face this problem. By adding the **note**, I want to solve this problem.

## Screenshots
Before going to 9th point user might face this problem which I explained above.

![image](https://user-images.githubusercontent.com/52650290/190509371-62bfaa6b-9955-495b-b43f-5a1195017384.png)

I want to solve like this

![image](https://user-images.githubusercontent.com/52650290/190509687-05a7e18f-9c1a-4f1f-bfc9-abbce713de5b.png)

### Desktop (please complete the following information):
 - OS: any
 - Browser: any
 - Version: any

### Smartphone (please complete the following information):
 - Device: any
 - OS: any
 - Browser any
 - Version any

## Additional context
N/A

***

#### 🎓 Add a tag to this issue for your current education role: **student**, **Student Ambassador** or **educator**.

***

#StudentAmbassador

**Comment by leestott** [comment]
@samipak458 Great feedback love the inclusion of the screenshots, feedback like this is excellent @softchris @saragibby

**Comment by samipak458** [comment]
Thank you @leestott sir 😊

**Comment by saragibby** [comment]
Yes, the detail and screenshots are extremly helpful to help us gt this addressed! 
This is a great find, and great feedback @samipak458! 🙌

**Comment by samipak458** [comment]
Thank you @saragibby ma'am 😊

**Comment by softchris** [comment]
reported in internal DevOps 78749
